resources/.tsan-ignore.txt

```diff
@@ -1,7 +1,7 @@
 # There are multiple data races identified in the uninstrumented libtbb (probably false positives)
 # However, be aware of concurrent_vector entries being potentially uninitialized:
 #   https://software.intel.com/en-us/blogs/2009/04/09/delusion-of-tbbconcurrent_vectors-size-or-3-ways-to-traverse-in-parallel-correctly
-race:^tbb::*
+race:tbb::*
```

Comment:
What's that?

Reply:
Not sure about this one, but I feel like less tbb false-positives popped up with this change.

Reply:
Can we at least have a comment explaining the actual line? I am wondering what the change actually does.